machineconfiguration/v1/types_pinnedimageset.go

```diff
@@ -0,0 +1,87 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=pinnedimagesets,scope=Cluster
+// +kubebuilder:subresource:status
+// +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/2198
+// +openshift:file-pattern=cvoRunLevel=0000_80,operatorName=machine-config,operatorOrdering=01
+// +openshift:enable:FeatureGate=PinnedImages
+// +kubebuilder:metadata:labels=openshift.io/operator-managed=
```

Comment:
Why does this label have an empty value?

Reply:
Digging through the history, it was added over 6 years ago in MCO. It was only added by Davids PR in API. It currently exists across other APIs in MCO like the general v1 types.go, MCN, PIS, MOSC, MOSB. If we do need this to change it would have to be a separate PR targeting all of the MCO APIs and not specifically this.
https://github.com/openshift/api/pull/1731
https://github.com/openshift/machine-config-operator/pull/1058

Reply:
If it's part of your tooling then of course it can stay